posthog/api/routing.py

```diff
@@ -157,6 +157,7 @@ def get_serializer_context(self) -> Dict[str, Any]:
         serializer_context["get_organization"] = lambda: self.organization
         return serializer_context
 
+    @lru_cache(maxsize=1)
```

Comment:
@Twixes just fyi, this function was borking your good work in making sure view.team_id & view.team don't make multiple requests to Team to get the same object.

Think this should still work as expected, but reduce team queries a bit more